contrib/olivere/elastic/elastictrace.go

```diff
@@ -117,5 +119,24 @@ func peek(rc io.ReadCloser, max int, n int) (string, io.ReadCloser, error) {
 	if err == io.EOF {
 		err = nil
 	}
+
+	// the snippet we just pulled out may be gzip encoded. if so, we need to
+	// decode it before we return so we can report the actual text to DataDog
+	if gzipped {
+		reader, err := gzip.NewReader(bytes.NewReader(snip))
```

Comment:
Since we've used short variable names in this function throughout, can we please do the same here? Suggestions: `rd`, `gzr`, `gzipr`